engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -328,7 +328,7 @@ public void setVignette(boolean vignette) {
     }
 
     public boolean isMotionBlur() {
-        return motionBlur && !isOculusVrSupport();
+        return motionBlur && !isVrSupport();
```

Comment:
There are two distinct categories of blur for what it is worth: motion blur (distorts the screen when you turn quickly, more visible if the system is laggy) and general blur (just uses distances to apply an overall blur to stuff, like when your vision intersects a goofy billboard plant in the foreground and everything behind it gets blurry)